s2spy/time.py

```diff
@@ -478,10 +478,19 @@ def _label_targets(self, intervals: pd.DataFrame) -> pd.DataFrame:
         Returns:
             pd.Dataframe: Dataframe with target periods labelled.
         """
-        return intervals.rename(
-            columns={i: f"(target) {i}" for i in range(self.n_targets)}
+        # rename preursors
+        intervals = intervals.rename(
+            columns={i: f"{self.n_targets - i - 1}" for i in range(self.n_targets,
+             len(intervals.columns))}
         )
 
+        # rename targets
+        intervals = intervals.rename(
+            columns={i: f"{self.n_targets - i}" for i in range(self.n_targets)}
+        )
+
+        return intervals
```

Comment:
Hi Yang, quick question; why don't we use an IntervalIndex here? I had changed this to having (string) index names to be able to label the targets, but that's not required anymore.

Reply:
Well spot! It is corrected now.